### DISCUSSION TITLE
Issue in reauth after upgrade

### DISCUSSION THREAD

**Discussion by jeremvv** [abraunegg/onedrive#1872] [Q&A] [answered]
Hi,
First, thanks for your great tool !

I'm using it for a while now with a dual account configuration (personnal onedrive and Office 365 pro onedrive) working perfectly.

I just had to change my Office 365 password and I couldn't handle the OneDrive account password change.

I realized my version was 2.4.13 so I upgraded it to 2.4.15 from source.

When I try to log with my Office 365 using --reauth, it keeps sending me back ".config/onedrive***/config/refresh_token: Not a directory" after i copy the URI.

I don't understand what I'm doing wrong.

onedrive --version returns:
`onedrive v2.4.15-28-gc4a7c1e`

Thank you for your feedback !

**Comment by jeremvv** [comment, marked as answer]
Sorry for my shitty question, the verbose option helped me to see the error. I was giving the config file with --confdir instead of the CONFIG DIRECTORY as written..... So it couldn't find my config file.

**Reply by abraunegg** [reply]
@jeremvv 
Correct - you need to use | add any other options into your configuration including specifying the correct options when using `--confdir`
